pkg/config/mesh/mesh.go

```diff
@@ -62,6 +62,7 @@ func DefaultMeshConfig() meshconfig.MeshConfig {
 	proxyConfig := DefaultProxyConfig()
 	return meshconfig.MeshConfig{
 		IngressClass:                      "istio",
+		IngressControllerMode:             meshconfig.MeshConfig_STRICT,
```

Comment:
@howardjohn Isn't the default value `MeshConfig_STRICT` set here?

Reply:
yes